src/themes/files/light/theme.json

```diff
@@ -148,9 +148,9 @@
   "typography": {
     "h1": {
       "fontFamily": "Inter",
+      "fontWeight": "600",
       "lineHeight": "32",
-      "fontSize": "24",
-      "fontWeight": "600"
+      "fontSize": "24"
```

Comment:
these values are intended as pixels?

Reply:
The theme is automatically generated by the `theme-generator.json` file 😬 we should update it properly by common agreement with the design team